tests/test_wave_03.py

```diff
@@ -141,12 +138,13 @@ def test_friends_unique_movies():
     friends_unique_movies = get_friends_unique_watched(amandas_data)
 
     # Arrange
-    assert len(friends_unique_movies) is 2
+    assert len(friends_unique_movies) is 3
+    assert {"title": "Title A"} in friends_unique_movies
     assert {"title": "Title D"} in friends_unique_movies
     assert {"title": "Title E"} in friends_unique_movies
 
 
-def test_friends_unique_movies_not_duplicated():
+def test_get_friends_unique_watched_returns_list_of_movies_amanda_has_not_watched_and_friends_have_with_only_one_friend():
```

Comment:
This and the test name on 185 ( `test_get_friends_unique_watched_returns_empty_list_when_amanda_has_seen_all_movies_their_friend_has_seen` ) seems excessively long.... they both are over [the recommended PEP-8 line length of 79 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).

Is there a different test name we could use?

Reply:
Also, I'm not necessarily a fan that the name references a local var in here.